services/shhext/service.go

```diff
@@ -253,39 +240,13 @@ func (s *Service) processReceivedMessages(messages []*whisper.Message) ([]dedup.
 			handler := PublisherSignalHandler{}
 			handler.DecryptMessageFailed(keyString)
 		default:
-			log.Error("Failed handling message with error", "err", err)
-		}
-	}
-
-	return dedupMessages, nil
-}
-
-func (s *Service) newSharedSecretHandler(filterService *filter.Service) func([]*sharedsecret.Secret) {
-	return func(sharedSecrets []*sharedsecret.Secret) {
-		var filters []*signal.Filter
-		for _, sharedSecret := range sharedSecrets {
-			chat, err := filterService.ProcessNegotiatedSecret(sharedSecret)
 			if err != nil {
-				log.Error("Failed to process negotiated secret", "err", err)
-				return
+				log.Error("Failed handling message with error", "err", err)
 			}
-
-			filter := &signal.Filter{
-				ChatID:   chat.ChatID,
-				SymKeyID: chat.SymKeyID,
-				Listen:   chat.Listen,
-				FilterID: chat.FilterID,
-				Identity: chat.Identity,
-				Topic:    chat.Topic,
-			}
-
-			filters = append(filters, filter)
-		}
-		if len(filters) != 0 {
-			handler := PublisherSignalHandler{}
-			handler.WhisperFilterAdded(filters)
```

Comment:
We don't need to inform status-react about this because we will keep all filters on status-go side, right? Instead, we will pass inform about messages through a signal?

Reply:
Correct, this is not needed anymore, it was kept there so it would be backward compatible, but code has been merged now in status-react so not needed